src/components/shared/FormField.js

```diff
@@ -1,16 +1,16 @@
-// import React from 'react'
+import React from 'react'
 
-// export const FormField = ({ fieldName, type, placeholder, register }) => {
-//     return (
-//         <input
-//             type={ type }
-//             {...register({ fieldName })}
-//             placeholder={ placeholder }
-//             className="
-//             w-full text-base px-4 py-2 border
-//             border-gray-300 rounded-lg
-//             focus:outline-none focus:border-atlas-400
-//             "
-//         ></input>
-//     )
-// }
\ No newline at end of file
+export const FormField = ({ fieldName, type, placeholder, register }) => {
+    return (
+        <input
+            type={ type }
+            {...register({ fieldName })}
+            placeholder={ placeholder }
+            className="
+            w-full text-base px-4 py-2 border
+            border-gray-300 rounded-lg
+            focus:outline-none focus:border-atlas-400
+            "
+        ></input>
+    )
+}
```

Comment:
should keep this commented out, as it's just for reference. might confuse others into using FormField component (does not work with React-hook-form for some reason if using this as a component)